### PR TITLE
Update menu on language preference change

### PR DESCRIPTION
**Pull request by ylavoie**
This fixes the menu portion of #950 by requiring an updated menu from the server on language preference change.

### CODE REVIEW

File: UI/js-src/lsmb/menus/Tree.js [hunk 1]
```diff
@@ -80,6 +84,41 @@ define([
...
+            this.model.constructor(dijit.byId(this.id).model);
+
+            // Rebuild the tree
+            this.postMixInProperties();
+            this._load();
+        },
+        startup: function () {
+            this.inherited(arguments);
+            var _this = this;
+            console.log(_this);
```
Comment (ehuelsmann) [inline UI/js-src/lsmb/menus/Tree.js:115]: console.log() ?

File: UI/js-src/lsmb/menus/Tree.js [hunk 2]
```diff
@@ -80,6 +84,41 @@ define([
...
+            connect.subscribe("lsmb/menus/Tree/refresh", function (message) {
+                /* eslint no-console:0 */
+                console.log("I got: ", message);
+                _this.refresh();
```
Comment (ehuelsmann) [inline UI/js-src/lsmb/menus/Tree.js:119]: same?

File: UI/users/preferences.html [hunk 3]
```diff
@@ -24,6 +24,17 @@
         <div data-dojo-type="dijit/layout/ContentPane" title="Preferences"
          data-dojo-props="selected:false">
  <form data-dojo-type="lsmb/Form" method="post" id="prefs" name="prefs" action="user.pl">
+  <script type="dojo/on" data-dojo-event="submit">
+    if(this.validate()){
+      /* eslint no-console:0 */
```
Comment (ehuelsmann) [inline UI/users/preferences.html:29]: no 'console' in sight?

Comment (ylavoie) [reply UI/users/preferences.html]: Fixed

File: UI/js-src/lsmb/menus/Tree.js (hunk 2 above)
Comment (ylavoie) [reply UI/js-src/lsmb/menus/Tree.js:116]: Fixed

**Pull request merged**